modules/prebidServerBidAdapter.js

```diff
@@ -545,6 +551,7 @@ const OPEN_RTB_PROTOCOL = {
       request.user = { ext: { digitrust: digiTrust } };
     }
 
+    aliases.ix = 'indexExchange';
```

Comment:
is there somewhere you can put this so that it only gets added if `ix` is involved in the PBS auction?

Reply:
Thanks for the suggestion; I pushed in a change that moved the alias registration.